doc/sphinx/source/recipes/recipe_emergent_constraints.rst

```diff
@@ -0,0 +1,147 @@
+Emergent constraints for equilibrium climate sensitivity
+========================================================
+
+Overview
+--------
+
+Calculates equilibrium climate sensitivity (ECS) versus
+
+1) southern ITCZ index, similar to fig. 2 from Tian (2015)
+2) lower tropospheric mixing index (LTMI), similar to fig. 5 from Sherwood et al. (2014)
+3) tropical mid-tropospheric humidity asymmetry index, similar to fig. 4 from Tian (2015)
+4) covariance of shortwave cloud reflection (Brient and Schneider, 2016)
+5) climatological Hadley cell extent (Lipat et al., 2017)
+
+The results are displayed as scatterplots.
+
+Note: this recipe requires pre-calulation of the equilibrium climate
```

Comment:
@axel-lauer 
> Not sure what you mean by scrambled documentation, though.

This note is not rendered properly, only the first line seems to be recognized as note.

Reply:
Thanks. Formatted note with directive .. note:: instead, looks fine now.